packages/utils/src/ts-eslint/Rule.ts

```diff
@@ -25,6 +25,11 @@ export interface RuleMetaDataDocs {
    * The URL of the rule's docs.
    */
   url?: string;
+
+  /**
+   * Mark this rule as feature = frozen.
```

Comment:
` = ` is not the same as `-` grammatically.

```suggestion
   * Mark this rule as feature-frozen.
```